### PR TITLE
[pulsar-io] hbase sink use BooleanSchema encode Boolean data

### PR DESCRIPTION
**Pull request by ambition119**
### Motivation

1. use pulsar client BooleanSchema encode Boolean data
2. remove unused code.

### Modifications

1. Use  BooleanSchema.encode() replace Bytes.toBytes()
2. remove  unused code, eg:  required = false,

### CODE REVIEW

File: pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java [hunk 1]
```diff
@@ -38,12 +36,9 @@
 @EqualsAndHashCode
 @ToString
 @Accessors(chain = true)
-public class HbaseAbstractConfig implements Serializable {
```
Comment (jiazhai) [inline pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java]: better to keep  Serializable

File: pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java [hunk 2]
```diff
@@ -38,12 +36,9 @@
...
-
-    private static final long serialVersionUID = 6783394446906640112L;
+public class HbaseAbstractConfig {
 
     @FieldDoc(
-        required = false,
```
Comment (jiazhai) [inline pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java]: :) why remove this `required` field?

File: pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java (hunk 1 above)
Comment (ambition119) [reply pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java]: think AbstractConfig not ser, extend class imp ser.

File: pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java (hunk 2 above)
Comment (ambition119) [reply pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java]: required = false is default value, can remove

**Comment by jiazhai** [comment]
retest this please

File: pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java (hunk 1 above)
Comment (ambition119) [reply pulsar-io/hbase/src/main/java/org/apache/pulsar/io/hbase/HbaseAbstractConfig.java]: done.

**Comment by ambition119** [comment]
retest this please

**Comment by jiazhai** [comment]
run cpp tests

**Pull request merged**